browser/ui/android/strings/android_brave_strings.grd

```diff
@@ -3932,6 +3932,27 @@ If you don't accept this request, VPN will not reconnect and your internet conne
     <message name="IDS_REWARDS_TOS_UPDATE_LINK_TEXT" desc="Terms of service text for rewards panel">
       See <ph name="TOS_START">&lt;tos&gt;</ph>Brave Rewards Terms of Service<ph name="TOS_END">&lt;/tos&gt;</ph>
     </message>
+    <message name="IDS_ENABLE_WINDOWS_TITLE" desc="Title for preference that enables or disables windows.">
+      Enable Windows
+    </message>
+    <message name="IDS_ENABLE_WINDOWS_SUMMARY" desc="Summary for preference that enables or disables windows.">
+      Open up to 5 windows at a time and move tabs from one window to another
+    </message>
+    <message name="IDS_MULTI_WINDOWS_DIALOG_TITLE" desc="Title for multiwindows detected dialog.">
+      Multiple Windows detected
```

Comment:
Needs consistent capitalization: either "Multiple windows detected" or "Multiple Windows Detected"

Reply:
My bad. I'll update it